tasks/src/audio-classification/about.md

```diff
@@ -50,10 +50,10 @@ data = query("sample1.flac")
 
 ### Speaker Identification
 
-Speaker Identification is classifying the audio for the person that is speaking in it. The speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.
+Speaker Identification consists of classifying the audio of the person speaking in it. Speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.
```

Comment:
I'd say:
Speaker Identification is of classifying the audio of the person speaking it. Speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.

Reply:
Imo the proposed change from Omar looks good